src/ReadLoader.cpp

```diff
@@ -386,39 +388,39 @@ void ReadLoader::processFastqFileThreadFunction(size_t threadId)
         SHASTA_ASSERT(scoresEnd > scoresBegin);
 
         // Check the header line.
-        if(headerEnd == headerBegin) {
+        if (headerEnd == headerBegin) {
             throw runtime_error("Empty header line for read at offset " +
-                to_string(headerBegin-fileBegin) + ".");
+                                to_string(headerBegin - fileBegin) + ".");
         }
-        if(*headerBegin != '@') {
+        if (*headerBegin != '@') {
             throw runtime_error("Read at offset " +
-                to_string(headerBegin-fileBegin) +
-                " does not begin with \"@\".");
+                                to_string(headerBegin - fileBegin) +
+                                " does not begin with \"@\".");
         }
 
         // Extract the read name.
         // It starts immediately following the '@' and ends at
         // first white space.
         readName.clear();
         const auto nameBegin = headerBegin + 1;
-        for(auto it=nameBegin; it!=headerEnd; ++it) {
+        for (auto it = nameBegin; it != headerEnd; ++it) {
             const char c = *it;
-            if(std::isspace(c)) {
+            if (std::isspace(c)) {
                 break;
             }
             readName.push_back(c);
         }
-        if(readName.empty()) {
+        if (readName.empty()) {
```

Comment:
I see lots of reformatting changes, many of which inconsistent with Shasta usage. Your editor is probably making them. Let's not worry about this for this time, but it would be best to avoid this in future PRs.